samples/features/azure-arc/configuration/Patch-ComputeMachineDisableFeatures

```diff
@@ -0,0 +1,128 @@
+$subscriptionId = ""
```

Comment:
@charmainewkchan should script be parameterized, as individual value checks are present already on top of script. 
something like :
param (
    [string]$subscriptionId,
    [string]$resourceGroupName,
    [bool]$whatIf = $false
)

Reply:
and RGName could be Parameter(Mandatory=$false) too..

Reply:
Sure thanks, updated